docs/cmdline-opts/create-dirs.d

```diff
@@ -5,7 +5,7 @@ Category: curl
 When used in conjunction with the --output option, curl will create the
 necessary local directory hierarchy as needed. This option creates the dirs
 mentioned with the --output option, nothing else. If the --output file name
-uses no dir or if the dirs it mentions already exist, no dir will be created.
+uses no dir or if the dirs it mentions already exist, no dirs will be created.
```

Comment:
I think this paragraph could gain from using the words spelled out as `directory` and `directories` instead of the abbreviations.

Reply:
Are you ok with force pushes? (Some repositories aren't and I've already been yelled at once here, so I'm going to be cautious before making significant tasks)